GUI/Main.cs

```diff
@@ -366,6 +366,13 @@ protected override void OnLoad(EventArgs e)
 
         protected override void OnFormClosing(FormClosingEventArgs e)
         {
+            //Only close the window, when the user has access to the "Exit" of the menu
+            if (!menuStrip1.Enabled)
```

Comment:
if the menustrip isn't enabled (e.g. the user isn't allowed to click on the menustrip), we disallow closing of the form.